src/containers/auth/LoginForm.tsx

```diff
@@ -1,9 +1,25 @@
 import Link from 'next/link';
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+import { rem } from 'polished';
 
 import { styled } from '../../lib/styles/stitches.config';
 
 const LoginForm = () => {
+  const [isMobile, setIsMobile] = useState(false);
+
+  const screenChange = (event: MediaQueryListEvent) => {
+    const { matches } = event;
+    setIsMobile(matches);
+  };
+
+  useEffect(() => {
+    const mobileWidthLarge = 640; // @mobileLarge
+    setIsMobile((window.innerWidth < mobileWidthLarge) ? true : false);
+    const mql = window.matchMedia('screen and (max-width: 640px)');
```

Comment:
```suggestion
    const mql = window.matchMedia(`screen and (max-width: ${mobileWidthLarge}px)`);
```
미디어 쿼리 테스트에 사용할 너비값도 앞에서 선언했던 변수를 사용하면 어떨까요?